### PR TITLE
Update domain.md

### PR DESCRIPTION
**Pull request by barclayn**
Adding a note at Frank's request

### CODE REVIEW

**Comment by barclayn** [comment]
#sign-off

**Comment by Danielabom** [comment]
Hi @barclayn, just checking, we do not have this note in beta, is that correct?

File: api-reference/v1.0/resources/domain.md [hunk 1]
```diff
@@ -27,6 +27,9 @@ To associate a domain with a tenant:
 
 5. [Configure](../api/domain-list-serviceconfigurationrecords.md) supported services by retrieving a list of records needed to enable services for the domain. Add the configuration record details to the domain's zone file using the domain registrar or DNS server configuration.
 
+>[!NOTE}
+> Setting the authentication type on a domain does not require the domain to be converted from managed to federated, or vice versa. Creating federation settings will automatically convert a domain from managed to federated, and deleting the federation settings will automatically convert the domain back to managed.
```
Comment (Danielabom) [inline api-reference/v1.0/resources/domain.md:31]:
```suggestion
> Setting the authentication type on a domain does not require the domain to be converted from managed to federated, or vice versa. When federation settings are created, the domain is automatically converted from managed to federated. On the other hand, deleting the federation settings will automatically convert the domain back to managed.
```

**Review by Danielabom — APPROVED** [review]
@barclayn, thank you for working on this PR. PR approved.

**Pull request closed without merging**